Jenkinsfile

```diff
@@ -79,6 +119,12 @@ pipeline {
 		}
 	}
 	post {
+		success {
+			recovery()
```

Comment:
Move the content of the `recovery` function here for clarity (it's only called here)